src/stores/useHolochainStore.js

```diff
@@ -9,28 +9,34 @@ import { hposHolochainCall } from '../services/hpos'
 
 const HC_APP_TIMEOUT = 35_000
 
+const __HC_LAUNCHER_ENV__ = "__HC_LAUNCHER_ENV__";
+const isLauncher = () => globalThis.window && __HC_LAUNCHER_ENV__ in globalThis.window;
+const getLauncherEnvironment = () => isLauncher() ? globalThis.window[__HC_LAUNCHER_ENV__] : undefined;
```

Comment:
```suggestion
```
Do we still need `getLauncherEnvironment`?  It seems like `isLauncher` is sufficient here now.